docker/Dockerfile-tools

```diff
@@ -84,14 +84,14 @@ RUN --mount=type=bind,target=. \
     CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} GOFLAGS="-mod=mod" go build -ldflags="${LD_FLAGS}" -a -o /out/kbcli cmd/cli/main.go
 
 # Use alpine
-FROM docker.io/alpine:3.17 as dist
+FROM docker.io/alpine:edge as dist
```

Comment:
`edge` is a word, like `latest`, it may be override by a new pushed image.
It's better to use tag [20230329](https://hub.docker.com/layers/library/alpine/20230329/images/sha256-79fa817edfa1565a7d176d576b136118e540fc20db6ae3d39c5fd1d4d5227a33?context=explore) to fix the image version.

Reply:
It will change to 3.19 after 3.19 released.

Reply:
will change to 20230329  tag